### DISCUSSION TITLE
Hardcoded text content in /layout/index.html

### DISCUSSION THREAD

**Discussion by romankupkovic** [thuliteio/doks#1045] [Q&A] [answered]
doks-child-theme

Is there a reason all this static text content is directly placed in /layouts/index.html ?

Should I just remove it and keep adding content from the Markdown pages?
<img width="801" alt="Screenshot 2023-04-19 at 16 56 26" src="https://user-images.githubusercontent.com/16864877/233023453-87b54226-0549-45fe-9572-fd4d7354497a.png">

**Comment by h-enk** [comment, marked as answer]
That's because you can make Doks completely your own using your own Bootstrap layouts/components. See e..g. [ScreenshotOne](https://screenshotone.com/)

However, I think it's a good idea to have some defaults for standard projects. I will put this on the list for Doks 1.0!

Note that, in Doks 0.5 content  in `content/en/_index.md` gets placed at the bottom (in the `sidebar-footer` part)— see also https://github.com/h-enk/doks-child-theme/blob/master/layouts/index.html#L51-L59
